changelog/README.md

```diff
@@ -0,0 +1,58 @@
+# Multi-file running changelog
+
+To avoid merge conflicts in the CHANGELOG.md file we accepted the multi-file running changelog strategy.
+
+To follow this strategy you should create a `.md` file for every PR. Choose a directory:
+
+- `changelog/unreleased/features` for **Features** changes
+- `changelog/unreleased/bugfixes` for **Bug fixes and improvements** changes
+- `changelog/unreleased/issues` for **Known issues :warning:** changes
+- `changelog/unreleased` for other changes
+
+You can use anything that allow .md format in changelog files.
+
+If you have implemented some features or bugfixes you should describe all of them:
+
+```
+- Description of changes in md format
+- Description of changes in md format also
+```
+
+You can choose any name for your changelog files because the GitHub action will rename files in
+`changelog/unreleased/features` and `changelog/unreleased/bugfixes` directories to `${PR_NUMBER}.md` when you open a PR.
```

Comment:
What about the issues directory and just `changelog/unreleased`?

Reply:
we need PR's links only for bugfixes and features